pkg/pool-weighted/contracts/smart/ManagedPool.sol

```diff
@@ -278,6 +350,23 @@ contract ManagedPool is BaseWeightedPool, ProtocolFeeCache, ReentrancyGuard {
         _startGradualWeightChange(startTime, endTime, _getNormalizedWeights(), endWeights, tokens);
     }
 
+    /**
+     * @dev Schedule a gradual swap fee change, from the current value to the given ending fee percentage,
+     * over startTime to endTime.
+     */
+    function updateSwapFeeGradually(
+        uint256 startTime,
+        uint256 endTime,
+        uint256 endSwapFeePercentage
+    ) external authenticate whenNotPaused nonReentrant {
```

Comment:
Do we want to give the option to set a `startSwapFeePercentage` here? It didn't make sense to do this for weights as you "never" want to change it instantaneously but it seems like it would be useful here.

Main example being where we're restarting trading and want to bump the swap fee up to 100% and let it decay back to its initial value, we currently need to make 2 calls to do this.